data/ThorOnTheRocks.json

```
@@ -0,0 +1,55 @@
+{
+  "name": "Gianluca Galota",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/ThorOnTheRocks.png",
+  "tags": ["javascript", "typescript", "react", "node", "open source", "design system"],
+  "socials": [
+    { "platform": "twitter", "url": "https://twitter.com/GianlucaGalota" },
+    { "platform": "github", "url": "https://github.com/ThorOnTheRocks" }
+  ],
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ThorOnTheRocks",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/GianlucaGalota",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
```

I recommend capitalizing the `i` here. What do you think?
```suggestion
      "name": "Follow me on LinkedIn",
```

---

Hey, @Panquesito7 those are personal preferences we can leave that. Might be improved later :)